content/applications/inventory_and_mrp/quality/quality_management/failure_locations.rst

```diff
@@ -0,0 +1,107 @@
+=================
+Failure locations
+=================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |QCPs| replace:: :abbr:`QCP (Quality Control Points)`
+
+In Odoo, quality control points (QCPs) are used to create quality checks, which prompt employees to
+confirm the quality of products, when they are included in certain operations. By setting one or
+more *failure locations* on a |QCP|, products that fail the quality checks it creates can be sent to
+one of the specified locations.
+
+.. important::
+   The *Failure Location* feature was added in version 17.0 of Odoo, and does not appear in any
+   previous version. To upgrade an Odoo database to a more recent version, see the documentation on
+   :doc:`database upgrades <../../../../administration/upgrade>`.
+
+Configuration
+=============
+
+To use failure locations, the *Storage Locations* setting must be enabled in the settings of the
+*Inventory* app. This setting allows for the creation of sub-locations within a warehouse, including
+failure locations.
+
+To enable the Storage Locations setting, navigate to :menuselection:`Inventory app --> Configuration
```

Comment:
```suggestion
To enable the *Storage Locations* setting, navigate to :menuselection:`Inventory app --> Configuration
```
suggesting the use of italics here because it is a proper name of a feature that would otherwise require a GUI (if it was currently being shown on-screen)